### PR TITLE
Bulgarian translation

### PR DESCRIPTION
**Pull request by miglen**

### CODE REVIEW

File: index_bg.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+pagecontent:
+  question: Какво представлява Homebrew?
+  what: Homebrew инсталира <a href="https://github.com/Homebrew/homebrew/tree/master/Library/Formula" title="Списък с пакети за Homebrew">всичко необходимо</a>, което Apple не може.
+  how: Homebrew инсталира пакетите в неговата директория и след това създава линкове към техните файлове в <code>/usr/local</code>.
+  prefix: Можете да инсталирате Homebrew където пожелаете. Той няма да инсталира файлове извън зададената директория.
+  createpackages: Създаването на Homebrew пакети е лесно и просто.
+  hack: Под капака е rubi и git, така че с лекота можете да правите промени.
+  formula: Homebrew формулите са прости Ruby скриптове:
+  editor: opens in $EDITOR!
```
Comment (bfontaine) [inline index_bg.html]: Didn’t you forget to translate this one?

Comment (miglen) [inline index_bg.html]: Nope, it was this way in some other languages like Russian, so I left it this way.

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Comment by miglen** [comment]
Thank you for your kind words @mikemcquaid if there is anything else I could translate to my native language I would be happy to do it.

**Pull request merged**